app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyDetailScreen.kt

```diff
@@ -68,49 +96,85 @@ fun SurveyDetailScreen(navigator: DestinationsNavigator, survey: Survey) {
             contentScale = ContentScale.Crop
         )
         HorizontalPager(
-            count = 1,
-            modifier = Modifier.padding(start = 20.dp, end = 20.dp, top = 108.dp),
-            userScrollEnabled = false
+            count = surveyQuestions.size + 1,
+            modifier = Modifier.padding(top = 108.dp),
+            state = pagerState
         ) { page ->
-            currentPage = page
             if (page == 0) {
-                SurveyDetailStartScreen(survey, modifier = Modifier.fillMaxWidth())
+                SurveyDetailStartScreen(survey, modifier = Modifier.fillMaxSize())
+            } else {
+                SurveyQuestionScreen(
+                    surveyQuestion = surveyQuestions[page - 1],
+                    pageNumber = page,
+                    totalNumberOfPage = surveyQuestions.size
+                )
             }
         }
         SurveyToolbar(
             navigator = navigator,
             modifier = Modifier
                 .fillMaxWidth()
-                .align(Alignment.TopCenter)
-                .padding(top = 57.dp, start = 8.dp, end = 15.dp),
+                .align(Alignment.TopStart)
+                .padding(top = 57.dp, start = 4.dp, end = 15.dp),
```

Comment:
No biggie since the layout looks good, but why is the `end` padding different from the `start`? 🤔